res/css/views/rooms/_TopUnreadMessagesBar.scss

```diff
@@ -25,19 +25,16 @@ limitations under the License.
 }
 
 .mx_TopUnreadMessagesBar::after {
-    content: "·";
+    content: "";
     position: absolute;
     top: -8px;
     left: 11px;
-    width: 16px;
-    height: 16px;
+    width: 4px;
+    height: 4px;
     border-radius: 16px;
-    font-weight: 600;
-    font-size: 30px;
-    line-height: 14px;
-    text-align: center;
-    color: $secondary-accent-color;
-    background-color: $accent-color;
+    overflow: hidden;
```

Comment:
Does the `overflow: hidden` actually have an effect here? I don't notice a difference toggling it in DevTools at least...

Reply:
Oh, no, I guess not. I had added this before I changed the dot from using text to using a border - having overflow:hidden fixed the hit box that was caused by the large font size. But now that we have `content: ""` there's nothing inside the box that could overflow.